### DISCUSSION TITLE
Is there a free african drum kit available for hydrogen ?

### DISCUSSION THREAD

**Discussion by fhenryco** [hydrogen-music/hydrogen#1756] [Hydrogen on Linux]
Hello

Is there a free african drum kit available for hydrogen ?
If not i also have a handsonic HPD-20 and i'm wondering if i could export its sound files (wave files) and then use them in hydrogen...
regards
Fred

**Comment by cme** [comment]
Hi! The `circAfrique v4` drumkit has djembes, dununs and kenkeni. You can install it by going to `Drumkits` -> `Online Import` and scrolling to find `circAfrique v4` in there. I don't know how that would compare!

**Comment by fhenryco** [comment]
Thank you. reading the doc, now i managed to understand how to import my own sounds in Hydrogen as wave files!
Dont know where i could find and test better african drum samples though.
